subprojects/distributions/src/integTest/groovy/org/gradle/DistributionIntegrationSpec.groovy

```diff
@@ -34,6 +34,7 @@ abstract class DistributionIntegrationSpec extends AbstractIntegrationSpec {
     @Rule public final PreconditionVerifier preconditionVerifier = new PreconditionVerifier()
 
     @Shared String version = GradleVersion.current().version
+    @Shared String baseVersion = GradleVersion.current().baseVersion.version
```

Comment:
That's fine, too, as long as we use the same expression everywhere. We use `versionBase` now in `groovyProject.gradle` and `baseVersion` here etc. Would be nice to clean up.